rpm/syslinux.spec

```diff
@@ -44,6 +44,8 @@ necessary to compile such modules.
 %package extlinux
 Summary: The EXTLINUX bootloader, for booting the local system
 Requires: syslinux
+Requires: coreutils
```

Comment:
You also should add Requires(post): as Requires: itself does not guarantee that the cmd's are there during %post

Reply:
If I add them in Requires(post) can I remove these "normal" Requires lines?

Reply:
Should be fine as it is only used there.

Reply:
fixed

Reply:
Ai đó có thể giúp tôi chuyển giấy phép từ xa2 sang xz2c được không dù đã xoá hết việc cài đặt trên xa2 và đăng xuất khỏi xa2 nhưng đăng nhập vào xz2c nó thiếu ứng dụng giấy phép